### ISSUE TITLE
Pretrained SCNN

### ISSUE THREAD

**Issue by mikeedi** [cardwing/Codes-for-Lane-Detection#2]
Thank you for code. If I understand correctly code you realized SCNN from https://arxiv.org/pdf/1712.06080.pdf in lane-detection-model folder. Do you have pretrained model for tensorflow? Can you share that?

**Comment by cardwing** [comment]
@mikeedi , the total training epochs are 9 w. Currently, I have not tested other learning rate schemes.

**Comment by chokkarapu** [comment]
@cardwing : Mentioning training episodes 9w means is it 9 Weeks ? is my interpretation right ?
Are you utilizing the GPUs for training?

**Comment by cardwing** [comment]
@chokkarapu , 90 k episodes. Besides, I use GPUs to accelerate training.

**Comment by cardwing** [comment]
Typically, it will take 2~3 days to train SCNN on my own machine (GeForce GTX TITAN X).

**Comment by cardwing** [comment]
@mikeedi , @chokkarapu , please use the latest code since I have found that using sgd+momentum can make the model converge much faster than pure sgd.

**Comment by chokkarapu** [comment]
> Typically, it will take 2~3 days to train SCNN on my own machine (GeForce GTX TITAN X).

Great....Thats good to hear...what is your GPU memory size in terms of GB?

**Comment by cardwing** [comment]
12 GB.
